.changeset/five-chicken-jam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/fondue": patch
+---
+
+update plate to v30
```

Comment:
```suggestion
chore: update plate to v30
```

And maybe a bit of text about the exposed stuff of plate which is now available via fondue